### DISCUSSION TITLE
Unlock Better Performance TM1963

### DISCUSSION THREAD

**Discussion by ravnized** [daliansky/XiaoMi-Pro#533] [Q&A]
Hi,
How can i make the changes written in the table?
Thanks in advance,
PS love your works man

**Comment by stevezz3** [comment]
I am afraid the table provided in [How to Unlock Better Performance](https://github.com/daliansky/XiaoMi-Pro-Hackintosh/tree/main/BIOS#how-to-unlock-better-performance-1) is only for TM1905. You might need FPTW64 to dump BIOS and find the corresponding offsets.
